lisa/analysis/rta.py

```diff
@@ -187,6 +235,30 @@ def tasks_window(self, task):
 
         return (start_time, end_time)
 
+    _PhaseWindow = namedtuple("PhaseWindow", ['id', 'start', 'end'])
```

Comment:
The type name should match its actual name:
```
     _PhaseWindow = namedtuple("_PhaseWindow", ['id', 'start', 'end']) 
```

Reply:
Let's keep it also public then...